src/main/java/mate/academy/springboot/datajpa/repository/ProductRepository.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.springboot.datajpa.repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    List<Product> findAllByPriceBetween(BigDecimal from, BigDecimal to);
+
+    @Query("from Product where category.id in :categoryId")
```

Comment:
It is better to write all SQL commands in upper case 
```suggestion
    @Query("FROM Product WHERE category.id IN :categoryId")
```